modules/testing/pages/integration/canceling.adoc

```diff
@@ -0,0 +1,14 @@
+= Canceling Integration pipelineruns for Pull Requests
+
+Pushing a new change to your PR creates a new build pipelinerun, followed by a new snapshot if build succeeded, then integration tests are triggered.
+In case you happen to push another change in your Pull Request, while integration tests pipelinerun are still running, the previous pipelinerun will be canceled.
+
+== How do I cancel the previous pipelinerun?
+
+If your integration tests are still running and you push new changes to your Pull Request, once build pipelinerun finishes with success and new snapshot is created,
+this change will be detected by integration service and will gracefully cancel previous integration test pipelinerun.
+
+== Which integration pipelinerun will be canceled?
```

Comment:
I feel this part can be combined with the last one since it has mentioned when older integration test pipelinerun is canceled.

Reply:
combined those two together